HatekonyCPP/Fraction.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <string>
+
+class Fraction
+{
+public:
+	static Fraction Parse(const std::string& value);
+
+public:
+	Fraction(int denominator);
```

Comment:
```suggestion
	Fraction(const int denominator);
```
Ugyanez minden bemeneti paraméternél.